doc/building.md

```diff
@@ -1458,16 +1458,24 @@ sh ./configure --with-jvm-variants=server \
 and run `make` normally.
 
 #### Building for Windows AArch64
+The Visual Studio Build Tools can be used for building the JDK without a full
+Visual Studio installation. To set up the Visual Studio 2022 Build Tools on a
+Windows AArch64 machine for a native build, launch the installer as follows
+in a Windows command prompt:
 
-Cygwin on a Windows AArch64 machine is an x86_64 environment. As a result, the
-build environment defaults to generating Windows x86_64 builds. To work around
-this behavior, use the `--build` option to generate a native Windows AArch64
-build:
+```
+vs_buildtools.exe --quiet --wait --norestart --nocache ^
+--installPath "%ProgramFiles(x86)%\Microsoft Visual Studio\2022\BuildTools" ^
+--add Microsoft.VisualStudio.Component.VC.CoreBuildTools ^
+--add Microsoft.VisualStudio.Component.VC.Tools.ARM64 ^
+--add Microsoft.VisualStudio.Component.Windows11SDK.22621
+```
+
+When installation is complete, configure the build using the same command used
+with a full Visual Studio installation:
 
 ```
-bash configure \
-  --with-boot-jdk=$BOOT_JDK \
-  --build=aarch64-unknown-cygwin
+bash configure --with-boot-jdk=$BOOT_JDK
```

Comment:
Probably lines 1474 - 1479 are now not needed any more because they just state you have to set the boot jdk but that's nothing specific for these builds.